compiler/modules/nback/build.bal

```diff
@@ -575,6 +575,11 @@ class Scaffold {
             builder.setCurrentDebugLocation(());
         }
     }
+
+    function unimplementedError(err:Message message) returns err:Unimplemented {
```

Comment:
Should this also be call `unimplementedErr` to match the other?

Reply:
fixed